<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginMenu</class>
 <widget class="QWidget" name="loginMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lbl_title">
     <property name="font">
      <font>
       <family>Helvetica Neue</family>
       <pointsize>24</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>781</width>
       <height>321</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>779</width>
        <height>334</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>779</width>
        <height>319</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_user5">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User 5</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_user6">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User 6</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_user1">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_user2">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_user3">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User 3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_user4">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>161</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Helvetica Neue</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>User 4</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="minimumSize">
        <size>
         <width>269</width>
         <height>20</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>266</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_newLogin">
       <property name="minimumSize">
        <size>
         <width>161</width>
         <height>81</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica Neue</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>New User</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>btn_back</zorder>
  <zorder>btn_newLogin</zorder>
  <zorder>lbl_title</zorder>
  <zorder>btn_newLogin</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
